SwiftTask/SwiftTask.swift

```diff
@@ -67,12 +67,12 @@ public class Task<Progress, Value, Error>: Cancellable, CustomStringConvertible
     public typealias RejectHandler = (Error) -> Void
     public typealias Configuration = TaskConfiguration
     
-    public typealias PromiseInitClosure = (fulfill: FulfillHandler, reject: RejectHandler) -> Void
-    public typealias InitClosure = (progress: ProgressHandler, fulfill: FulfillHandler, reject: RejectHandler, configure: TaskConfiguration) -> Void
+    public typealias PromiseInitClosure = (_ fulfill: FulfillHandler, _ reject: RejectHandler) -> Void
+    public typealias InitClosure = (_ progress: ProgressHandler, _ fulfill: FulfillHandler, _ reject: RejectHandler, _ configure: TaskConfiguration) -> Void
     
     internal typealias _Machine = _StateMachine<Progress, Value, Error>
     
-    internal typealias _InitClosure = (machine: _Machine, progress: ProgressHandler, fulfill: FulfillHandler, _reject: _RejectInfoHandler, configure: TaskConfiguration) -> Void
+    internal typealias _InitClosure = (_ machine: _Machine, _ progress: ProgressHandler, _ fulfill: FulfillHandler, _ _reject: _RejectInfoHandler, _ configure: TaskConfiguration) -> Void
```

Comment:
So we now need to erase argument labels for function types... 🤔

- [[SE-0111] Remove type system significance of function argument labels](https://github.com/apple/swift-evolution/blob/master/proposals/0111-remove-arg-label-type-significance.md)
- https://github.com/apple/swift/commit/ca440c615f01a1f2067cdd4f10327f3da81b2b96